prowler/providers/aws/services/cognito/cognito_service.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.lib.service.service import AWSService
+
+
+################## Cognito
+class Cognito(AWSService):
+    def __init__(self, audit_info):
+        super().__init__("cognito-idp", audit_info)
+        self.user_pools = []
+        self.__threading_call__(self.__list_user_pools__)
+        self.__threading_call__(self.__describe_user_pools__)
+        self.__threading_call__(self.__get_user_pool_mfa_config__)
+
+    def __list_user_pools__(self, regional_client):
+        logger.info("Cognito - Listing User Pools...")
+        try:
+            user_pools_paginator = regional_client.get_paginator("list_user_pools")
+            for page in user_pools_paginator.paginate(MaxResults=60):
+                for user_pool in page["UserPools"]:
+                    arn = f"arn:{self.audited_partition}:cognito-idp:{regional_client.region}:{self.audited_account}:userpool/{user_pool['Id']}"
+                    if not self.audit_resources or (
+                        is_resource_filtered(arn, self.audit_resources)
+                    ):
+                        try:
+                            self.user_pools.append(
+                                UserPool(
+                                    id=user_pool["Id"],
+                                    arn=arn,
+                                    name=user_pool["Name"],
+                                    region=regional_client.region,
+                                    last_modified=user_pool["LastModifiedDate"],
+                                    creation_date=user_pool["CreationDate"],
+                                    status=user_pool.get("Status", "Disabled"),
+                                )
+                            )
+                        except Exception as error:
+                            logger.error(
+                                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+                            )
+        except Exception as error:
+            logger.error(
+                f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
+            )
+
+    def __describe_user_pools__(self, regional_client):
+        logger.info("Cognito - Describing User Pools...")
+        try:
+            for user_pool in self.user_pools:
+                try:
+                    user_pool_details = regional_client.describe_user_pool(
+                        UserPoolId=user_pool.id
+                    )
+                    user_pool.password_policy = PasswordPolicy(
+                        **user_pool_details.get("Policies", {}).get(
+                            "PasswordPolicy", {}
+                        )
```

Comment:
This is not getting the values configured in the AWS Console, please check it out and also the `ERROR` logs.